README.md

```diff
@@ -1,3 +1,59 @@
 # Castle Crashers
 
-A CMPT-306 game made by a team of 6 people in the span of 1.5ish months!
\ No newline at end of file
+A CMPT-306 dungeon crawler game built in Unity and C# by a team of 7 people in the span of 1.5ish months!
+The name was a coincidence; we just like the ring to it (so did The Behemoth for the real Castle Crashers!)
+Note: the class focused on Game Mechanics, most of the visual and audio assets were used from free libraries or purchased from the Unity Store.
+
+## Overview
+The goal of the game is to traverse deeper and deeper to new levels of the castle, wield a variety of weapons to battle a variety of enemies, utilize gathered loot and coins to get stronger via opening chests, recieving permanent power upgrades, and buying stat & skill upgrades. Once the player reach the end of their journey, usually ended from a gruesome death, they would recieve a score based on their performance in the game.
+Link to our game trailer:
+https://youtu.be/27xrTJ3nenE
+
+### Stats
+Stats could be upgraded between levels and affected the player power in different ways:
+- Strength: increases damage of weapons. 
+- Agility: increases the attack and movement speed of the player.
+- Stamnia: increases player health and the effectiveness of health potions.
+
+### Weapons
+Several weapons were available to the player, most weapons types focusing on different ranged weapons:
+- Sword: a basic weapon that would sweep and do damage in a small range from the player.
+- Rock: a thrown weapon that is easy to target and hit with that does moderate damage.
+- Arrow: a thrown weapon that hits enemies for a moderate amount of damage and leaves a small bleed DoT (damage of time) effect on enemies.
+- Fireball: a small, but hard to hit with, fireball that does a small amount of initial damage but leaves enemies on fire; the on fire DoT ticks for heavy fire damage.
```

Comment:
I'd suggest changing "the on fire DOT ticks for heavy fire damage" to "while the enemy is on fire, they periodically take heavy fire damage"